File: pinyougou-web/pinyougou-manager-web/src/main/webapp/js/controller/indexController.js
app.controller('indexController',function ($scope,baseService) {
    //获取用户的的数据
    $scope.showLoginName=function () {
        baseService.sendGet("/showLoginName").then(function (value) {
            $scope.loginName=value.data.loginName;
        })
    }
})

File: pinyougou-web/pinyougou-shop--web/src/main/java/com/pinyougou/shop/controller/IndexController.java
package com.pinyougou.shop.controller;

import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.HashMap;
import java.util.Map;

@RestController
@RequestMapping
public class IndexController {

    @GetMapping("showLoginName")
    public Map<String,String> showLoginName(){
        try {
            String name = SecurityContextHolder.getContext().getAuthentication().getName();
            Map<String , String> map = new HashMap<>();
            map.put("loginName",name);
            return map;
        } catch (Exception e) {
            throw new RuntimeException(e);
        }
    }
}
